.github/workflows/on-main-branch-commit-push-liminal-node-to-ecr.yml

```diff
@@ -0,0 +1,83 @@
+---
+name: Builds and pushes aleph-node liminal to ECR on push to main branch
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  build-liminal-node:
+    name: Build liminal node
+    uses: ./.github/workflows/_build-liminal-node.yml
+    secrets: inherit
+
+  push-liminal-node-image-to-ecr:
+    name: Push liminal node docker image to the ECR
+    needs: [build-liminal-node]
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout aleph-node sources
+        uses: actions/checkout@v3
+
+      - name: Call action get-ref-properties
+        id: get-ref-properties
+        uses: ./.github/actions/get-ref-properties
+
+      - name: Download node production binary from artifacts
+        uses: actions/download-artifact@v2
+        with:
+          name: aleph-node-liminal
+          path: target/release/
+
+      - name: Build production docker node image
+        id: build-image
+        run: |
+          chmod +x target/release/aleph-node
+          docker build --tag aleph-node:latest -f ./docker/Dockerfile .
```

Comment:
Later on you have `CURRENT_IMAGE: aleph-node-liminal` but here you're using a non-liminal tag. It should probably be `--tag aleph-node-liminal:latest`.

Reply:
good catch